examples/gno.land/r/samourai/samdao/messages.gno

```diff
@@ -0,0 +1,277 @@
+	s := ""
+	s += ufmt.Sprintf("New prod project: %s\n", m.Name)
+	s += ufmt.Sprintf("Description: %s\n", m.Description)
+	s += ufmt.Sprintf("Members:\n")
+	for _, member := range m.Members {
+		s += ufmt.Sprintf("  - %s\n", member.String())
+	}
+	s += ufmt.Sprintf("Tasks:\n")
+	for _, task := range m.Tasks {
+		s += ufmt.Sprintf("  - %s\n", task.Name)
+		s += ufmt.Sprintf("    - Description: %s\n", task.Description)
+		s += ufmt.Sprintf("    - Status: %s\n", task.Status)
+	}
+	return s
+}
+
+func NewNewProdProjectHandler(pmTarget *project_manager.ProjectManager) daokit.MessageHandler {
+	return daokit.NewMessageHandler(MsgNewProdProjectKind, func(ipayload interface{}) {
+		msg, ok := ipayload.(*MsgNewProdProject)
+		if !ok {
+			panic(errors.New("invalid payload type"))
+		}
+		pmTarget.AddProject(msg.Name, msg.Description, project_manager.DivisionProd, msg.Members, msg.Tasks)
+	})
+}
+
+func NewNewProdProjectMsg(name string, description string, tasks []project_manager.Task, members []std.Address) daokit.ExecutableMessage {
+	return daokit.NewMessage(MsgNewProdProjectKind, &MsgNewProdProject{Name: name, Description: description, Tasks: tasks, Members: members})
+}
+
+// NEW TECH PROJECT
+
+const MsgNewTechProjectKind = "gno.land/r/samourai/samdao.NewTechProject"
+
+type MsgNewTechProject struct {
+	Name        string
+	Description string
+	Tasks       []project_manager.Task
+	Members     []std.Address
+}
+
+func (m *MsgNewTechProject) String() string {
+	s := ""
+	s += ufmt.Sprintf("New tech project: %s\n\n", m.Name)
+	s += ufmt.Sprintf("Description: %s\n\n", m.Description)
+	s += ufmt.Sprintf("Members:\n\n")
+	for _, member := range m.Members {
+		s += ufmt.Sprintf("  - %s\n\n", member.String())
+	}
+	s += ufmt.Sprintf("Tasks:\n\n")
+	for _, task := range m.Tasks {
+		s += ufmt.Sprintf("  - %s\n\n", task.Name)
+		s += ufmt.Sprintf("    - Description: %s\n\n", task.Description)
+		s += ufmt.Sprintf("    - Status: %s\n\n", task.Status)
+	}
+	return s
+}
+
+func NewNewTechProjectHandler(pmTarget *project_manager.ProjectManager) daokit.MessageHandler {
+	return daokit.NewMessageHandler(MsgNewTechProjectKind, func(ipayload interface{}) {
+		msg, ok := ipayload.(*MsgNewTechProject)
+		if !ok {
+			panic(errors.New("invalid payload type"))
+		}
+
+		pmTarget.AddProject(msg.Name, msg.Description, project_manager.DivisionTech, msg.Members, msg.Tasks)
+	})
+}
+
+func NewNewTechProjectMsg(name string, description string, tasks []project_manager.Task, members []std.Address) daokit.ExecutableMessage {
+	return daokit.NewMessage(MsgNewTechProjectKind, &MsgNewTechProject{Name: name, Description: description, Tasks: tasks, Members: members})
+}
+
+// EDIT PAYROLLS
+
+const MsgNewPayrollsKind = "gno.land/r/samourai/samdao.NewPayrolls"
+
+type MsgNewPayrolls struct {
+	beneficiary std.Address
+	label       string
+	distrib     payrolls.DistribFn
+	breakup     payrolls.BreakupFn
+}
+
+func (m *MsgNewPayrolls) String() string {
```

Comment:
Might be better to expose `NewMonthly` instead of raw `NewPayroll` with exposed distrib functions